### ISSUE TITLE
Telnyx gone

### ISSUE THREAD

**Issue by repunset274** [0perationPrivacy/VoIP#136]
Hi everyone,

With MB's recent disavowal of Telnyx [(podcast link)](https://inteltechniques.com/blog/2022/04/22/the-privacy-security-osint-show-episode-258/) as a voip provider, are the devs open to the idea of implementations allowing for API connection to other providers, such as voip.ms or flowroute?

**Comment by 0perationPrivacy** [comment]
Yes, looking into voip.ms, they have an API for integration.

Open to all other suggestions.
Only condition is that the account creation should be fairly anonymous (VPN, privacy credit card, alias email, voip number usage for creation). Also a KYC check with a passport or ID is unacceptable.

**Comment by 0perationPrivacy** [comment]
So, voip.ms is a no go. I created an account with simplelogin and protonvpn on, got flagged immediately for manual review. Chatted with support and they refuse to give me a reason and simply deny service.

![voip ms chat](https://user-images.githubusercontent.com/89225730/171565903-e409e459-eabb-4b4a-a26a-83fb4eacfcd4.png)

**Comment by xetch** [comment]
Flowroute won't be much better tbh. They have a very finicky fraud system in my experiences.

**Comment by repunset274** [comment]
That is unfortunate. In some fairness, it will be very hard to avoid all scrutiny with any provider, all of them have anti-fraud measures to some degree, and most can be worked around. But I also agree with the sentiment that we should not have to resort to grayscale measures just to maintain privacy while using a service.

**Comment by ad4g472aqvti** [comment]
@0perationPrivacy SignalWire is another consideration, although it's not perfect either. 2FA is only email-based, and there's one complication when setting up an account. While sign up can be done with a VPN, alias name, and email, adding a card cannot be done with a VPN. SignalWire support told me the purchase has to be done with no VPN. So going on public Wi-Fi, adding the card, making a purchase with auto recharge makes using SignalWire possible.

**Comment by monkeysac** [comment]
> SignalWire is another consideration, although it's not perfect either.

Though im a long-time user of Signalwire i cannot recommend them for integration here. Unfortunately they're on board with the campaign registration cr*p STIR/SHAKEN brought about, as many have been.

**Comment by ad4g472aqvti** [comment]
> Though im a long-time user of Signalwire i cannot recommend them for integration here. Unfortunately they're on board with the campaign registration cr*p STIR/SHAKEN brought about, as many have been.
Yeah STIR/SHAKEN has been a recent complication with using VOIP for me. Although SignalWire doesn't offer the highest attestation (B level if you don't wanna go through invasive registration), it's still an improvement over Twilio which gives C level with no registration.

**Comment by jkelly99** [comment]
Just fyi, Signalwire has an open $500 reward for any opensource project build with their SDK. Since they seem to have a SDK that's pretty compatible with Twilio's, maybe a fairly easy $500 for this project?
[https://signalwire.com/blogs/product/open-source-contribution-reward](url)

**Comment by ad4g472aqvti** [comment]
I tried Flowroute a couple hours ago. Can immediately say they are out. Locked my account even with a custom domain and they want an ID WITH proof of address (utility bill or bank statement).

SignalWire implementation might just be the least path of resistance far as getting support for another VOIP provider goes due to similar API to twilio and somewhat simple registration.

**Comment by Hub-ert** [comment]
Vonage looks promising.
